src/interface.jl

```diff
@@ -514,31 +510,27 @@ transformed(d) = transformed(d, bijector(d))
 
 Returns the constrained-to-unconstrained bijector for distribution `d`.
 """
+bijector(d::Distribution) = DistributionBijector(d)
 bijector(d::Normal) = IdentityBijector
 bijector(d::MvNormal) = IdentityBijector
 bijector(d::PositiveDistribution) = Log()
 bijector(d::MvLogNormal) = Log()
 bijector(d::SimplexDistribution) = simplex_b_proj
+bijector(d::KSOneSided) = Logit(zero(eltype(d)), zero(eltype(d)))
 
-_union2tuple(T1::Type, T2::Type) = (T1, T2)
-_union2tuple(T1::Type, T2::Union) = (T1, _union2tuple(T2.a, T2.b)...)
-_union2tuple(T::Union) = _union2tuple(T.a, T.b)
+const BoundedDistribution = Union{
+    Arcsine, Biweight, Cosine, Epanechnikov, Beta, NoncentralBeta
+}
+bijector(d::BoundedDistribution) = Logit(minimum(d), maximum(d))
 
-bijector(d::KSOneSided) = Logit(zero(eltype(d)), zero(eltype(d)))
-for D in _union2tuple(UnitDistribution)
-    # Skipping KSOneSided because it's not a parametric type
-    if D == KSOneSided
-        continue
-    end
-    @eval bijector(d::$D{T}) where {T<:Real} = Logit(zero(T), one(T))
-end
+const LowerboundedDistribution = Union{Pareto, Levy}
+bijector(d::LowerboundedDistribution) = Log() ∘ Shift(- minimum(d))
 
 # FIXME: (TOR) Can we make this type-stable?
```

Comment:
Do we need this comment still?

Reply:
I've left it there for `Truncated`; I'd say leave it so that in the case someone has an idea OR for some reason whether or not a distribution is bounded becomes available at compile-time, this is a reminder to revisit the problem:)

Reply:
I see.